cluster-scope/overlays/nerc-ocp-test/nodefeaturediscoveries/nfd-instance.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-instance
+  namespace: openshift-nfd
+spec:
+  customConfig:
+    configData: ""
+  instance: ""
+  operand:
+    image: registry.redhat.io/openshift4/ose-node-feature-discovery:v4.11
+    imagePullPolicy: Always
+    servicePort: 0
```

Comment:
servicePort 0 means dynamically assigned?

Reply:
That would be my guess. From `oc explain nodefeaturediscovery.spec.operand.servicePort`:
```
ServicePort specifies the TCP port that nfd-master listens for incoming requests.
```